content/en/docs/setup/platform-setup/kops/index.md

```diff
@@ -9,7 +9,7 @@ test: no
 ---
 
 {{< tip >}}
-This is no longer required for kops 1.22+.
+This is no longer required for Kops 1.22 or higher.
```

Comment:
something more positive like "No special configuration is required to run Istio on clusters managed by [Kops](https://kops.sigs.k8s.io/) since version 1.22.  For prior versions, you will need to ..."